advocacy_docs/partner_docs/Cyberark/04-Configuratingcyberark.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Configuration'
+description: 'Walkthrough on configuring the integration'
+---
+
+## Prerequisites
+
+Implementing CyberArk CPM (Central Policy Manager) with EDB Postgres Advanced Server requires the following components:
+
+## Configuring your EDB Postgres Advanced Server
+
+Following component is needed before integrating EDB Postgres Advanced Server with CyberArk CPM (Central Policy Manager):
```

Comment:
Would reword:
 The following component is needed before integrating EDB Postgres Advanced Server with CyberArk CPM (Central Policy Manager):

Reply:
done